packages/aws-cdk-lib/aws-eks/lib/helm-chart.ts

```diff
@@ -108,10 +108,21 @@ export class HelmChart extends Construct {
    * The CloudFormation resource type.
    */
   public static readonly RESOURCE_TYPE = 'Custom::AWSCDK-EKS-HelmChart';
+  public readonly chart: string | undefined;
+  public readonly repository: string | undefined;
+  public readonly version: string | undefined;
+  public readonly chartAsset: Asset | undefined;
```

Comment:
I think it's better to match the [HelmChartOptions](https://github.com/aws/aws-cdk/blob/7af20babac091dfa39bbeb625069799c0f002a76/packages/aws-cdk-lib/aws-eks/lib/helm-chart.ts#L11) interface declaration a change these to:
```
public readonly chart?: string;
...
```

Reply:
Will do